requirements.txt

```
@@ -9,15 +9,15 @@ cryptography==41.0.7
 Cython==3.0.7
 dill==0.3.6
 future==0.18.3
-idna==3.3
+idna==3.6
 mpmath==1.3.0
 networkx==3.1
 ninja==1.11.1
 ntlm-auth==1.5.0
 numpy==1.22.3
 orjson==3.6.8
 pbr==6.0.0
-pennylane==0.30.0
+pennylane==0.33.1
```

```suggestion
pennylane==0.34
```

You should bump the lightning version too.